src/eventlog/models.py

```diff
@@ -19,7 +19,7 @@
 # A user session
 class LoginSession(models.Model):
     id = models.CharField(primary_key=True, default=uuid4, max_length=36)
-    user = models.ForeignKey(to=ClusiveUser, on_delete=models.PROTECT)
+    user = models.ForeignKey(to=ClusiveUser, on_delete=models.SET_NULL, null=True)
```

Comment:
Do you have un-checked-in migration files for these model changes?
Clean build is still giving me the not-null constraint.

Reply:
I remember having to make migrations after those model changes, but I cannot find the migration files locally.  A possibility is that this occurred around the time my laptop crashed, and I had rebuild it from backups.  It's possible that the migration file(s) were lost at that time.  Any suggestions on how to recreate them?